djangobaseview/static/js/guestbook/view/GuestBookView.js

```diff
@@ -0,0 +1,62 @@
+define([
+    "dojo/dom",
+    "dojo/on",
+    "dojo/request",
+    "dojo/cookie",
+    "dojo/_base/declare",
+    "dojo/_base/array",
+    "./_ViewBaseMixin",
+    "./GreetingView",
+    "dojo/text!../templates/GuestBookView.html",
+    "dijit/form/Form",
+    "dijit/form/TextBox",
+    "dijit/form/Textarea",
+    "dijit/form/Button",
+], function (dom, on, request, cookie, declare, array, _ViewBaseMixin, GreetingView, template) {
+    return declare("guestbook.view.GuestBookView", [_ViewBaseMixin, GreetingView], {
+        templateString: template,
+        //	some properties
+        baseClass: "someWidget",
+
+        postCreate: function () {
+            // do my stuff, then...
+            this.inherited(arguments);
+
+            // add guestbook api
+            var form = dom.byId('signForm');
+            on(form, "submit", function (e) {
```

Comment:
hãy đặt `on()` vào trong `this.own()`, để sự kiện được tự động remove khi widget bị destroy

http://dojotoolkit.org/reference-guide/1.10/dijit/Destroyable.html